Core/DateTimeHelpers.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Core
+{
+    public static class DateTimeHelpers
+    {
+        public static Date ToDate(this DateTime dt) => new Date(dt);
+    }
+}
```

Comment:
Codacy found an issue: [Add a new line at the end of the file 'DateTimeHelpers.cs'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=6077033)